keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -64,4 +71,18 @@ class ApplicationController(
       "hasManagedResources" to resourceRepository.hasManagedResources(application),
       "hasDeliveryConfig" to hasDeliveryConfig)
   }
+
+  @PostMapping(
+    path = ["/{application}/pause"]
+  )
+  fun pause(@PathVariable("application") application: String) {
+    resourcePauser.pauseApplication(application)
+  }
+
+  @PostMapping(
+    path = ["/{application}/resume"]
+  )
+  fun resume(@PathVariable("application") application: String) {
+    resourcePauser.resumeApplication(application)
+  }
```

Comment:
This is minor, but I feel like REST semantics would suggest these were a PUT and a DELETE request on the `/{application}/pause` endpoint (you're putting or removing a pause flag on the application).

Reply:
Oh sure that makes sense, I'll update